scripts/builtin/tSNE.dml

```diff
@@ -72,6 +106,13 @@ m_tSNE = function(Matrix[Double] X, Integer reduced_dims = 2, Integer perplexity
     sumW = rowSums(W)
     g = Y * sumW - W %*% Y
     dY = momentum*dY - lr*g
+
+    norm = sum(dY^2)
+    if(is_verbose & itr %%10 ==0){
```

Comment:
maybe add a print iteration variable, to indicate how frequent the logging is and default it to 10.

Reply:
Should this be a new input parameter or just a static variable in the script?

Reply:
input variable.